docs/getting-started/DigitalOcean.md

```diff
@@ -120,7 +120,7 @@ sudo iptables -A PREROUTING -t nat -i eth0 -p tcp --dport 80 -j DNAT  --to 192.1
 ```
 
 You should now be able to access the containers using the public IP address of your DigitalOcean host on port 80 by 
-visiting http://<host public ip>:80 or running:
+visiting http://`<host public ip>`:80 or running:
```

Comment:
I was adopting the style used in one of the other Getting Started docs. As it currently stands, those angle brackets are swallowed as an HTML tag. Feel free to change, but there's at least one other doc (not in this commit) where that style is used.